### PR TITLE
docs: update examples in valid section

### PR DESCRIPTION
**Pull request by skanehira**
fix: https://github.com/denoland/deno_lint/issues/1221
ref: https://github.com/denoland/deno_lint/pull/1222

### CODE REVIEW

File: docs/rules/no_deprecated_deno_api.md [hunk 1]
```diff
@@ -77,28 +77,25 @@ function foo(file: Deno.File) {
...
-const b = await readAll(reader);
-const c = readAllSync(reader);
-
-// write
-import { writeAll, writeAllSync } from "https://deno.land/std/io/util.ts";
-await writeAll(writer, data);
-writeAllSync(writer, data);
+// writeAll
+// reader is `ReadableStream` and writer is `WritableStream`
+const reader = ReadableStream.from(...);
```
Comment (magurotuna) [inline docs/rules/no_deprecated_deno_api.md:87]: It'd be better to keep the sample code valid as JS/TS code. What I'd do instead:

```suggestion
const reader = ReadableStream.from([1, 2, 3]);
```

**Review by magurotuna — COMMENTED** [review]
Nice fix! Left one comment

**Comment by magurotuna** [comment]
Can you run the commands as described in the step 6 here? https://github.com/denoland/deno_lint#submitting-a-pull-request

**Comment by skanehira** [comment]
> Can you run the commands as described in the step 6 here? https://github.com/denoland/deno_lint#submitting-a-pull-request

@magurotuna Fixed :)

**Review by magurotuna — APPROVED** [review]
LGTM, thanks!

**Pull request merged**